src/dmd/backend/cdef.d

```diff
@@ -605,6 +605,7 @@ enum
     CFG2expand      = 0x4000,  // expanded output to list file
     CFG2stomp       = 0x8000,  // enable stack stomping code
     CFG2gms         = 0x10000, // optimize debug symbols for microsoft debuggers
+    CFG2genmain     = 0x20000, // main entrypoint is generated
```

Comment:
I don't know the meaning of each flag enum, so I added to what I saw more appropriate. If there is any meaning to it, please tell me to add proper documentation to it.